torchaudio/prototype/tacotron2.py

```diff
@@ -1105,3 +1163,60 @@ def infer(self, text: Tensor, text_lengths: Tensor) -> Tuple[Tensor, Tensor, Ten
         alignments = alignments.unfold(1, n_batch, n_batch).transpose(0, 2)
 
         return mel_outputs_postnet, mel_specgram_lengths, alignments
+
+
+def tacotron2(checkpoint_name: str) -> Tacotron2:
+    r"""Get pretrained Tacotron2 model.
+
+    Args:
+        checkpoint_name (str): The name of the checkpoint to load. Available checkpoints:
```

Comment:
I just realized that there is no documentation for Tactoron2. Can you add that so that we can see how this docstring is converted to documentation?

Reply:
I'll address this in a separate PR.
Thanks.